pass.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <title>Sprawdzanie hasła</title>
+    <meta charset="UTF-8">
+    <script type="text/javascript">
+        function test_str() {
```

Comment:
w 1/3 działa :D
Strasznie ten JS jest nielogiczny jeszcze XD Co powinnam zrobić żeby 
1. Wyświetliły mi się te ostrzeżenia na czerwono (wiem, ze muszę je dodać jeszcze, średnio mam pomysł jak, czy mam milion ifów robić?)
2.Sprawdziło mi czy a===b (chodzi  o to, ze nie chce dublować funkcji test_str() ajak próbowałam podstawić x to mi nie działało XD

Reply:
1. Ustaw w stylach kolor na czerwony ;) Będziesz musiała też zmieniać widoczność tych ostrzeżeń oczywiście. Z poziomu javascriptu możesz zmieniać wszystkie atrybuty elementów w htmlu. Wśród atrybutów znajduje się jeden odpowiedzialny za widoczność elementu - nazywa się hidden i przyjmuje wartość typu true/false. Możesz umieścić komunikat w jakmś znaczniku html i zmieniać jego widoczność w zależności od wprowadzonego tesktu, np. dla wiadomości związanej ze zbyt krótkim hasłem możesz utworzyć następujący znaacznik: `<div style="color:red" id="too-short-passwd-msg">Za krótkie hasło. Min. 8 znaków.</div>` A w javascripcie by zmienić jego widoczność (w zależności od długości hasła) możesz wpisać: `document.getElementById("too-short-passwd-msg").hidden=true`
2. Nie rozumiem chyba tego :(